Src/java/elm-fhir/src/test/resources/org/cqframework/cql/elm/requirements/fhir/CMS986/resources/library-Measure-Observation-1-requirements.json

```diff
@@ -0,0 +1,196 @@
+  }, {
+    "type": "depends-on",
+    "display": "Value set Malnutrition Risk Screening",
+    "resource": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1095.92"
+  }, {
+    "type": "depends-on",
+    "display": "Value set Observation Services",
+    "resource": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1111.143"
+  }, {
+    "type": "depends-on",
+    "display": "Value set Emergency Department Visit",
+    "resource": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.3.117.1.7.1.292"
+  }, {
+    "type": "depends-on",
+    "display": "Value set Malnutrition Screening Finding of Not At Risk Result",
+    "resource": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1095.34"
+  }, {
+    "type": "depends-on",
+    "display": "Value set Malnutrition Screening Finding of At Risk Result",
+    "resource": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1095.89"
+  }, {
+    "type": "depends-on",
+    "display": "Value set Dietitian Referral",
+    "resource": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1095.91"
+  } ],
+  "parameter": [ {
+    "name": "Measurement Period",
+    "use": "in",
+    "min": 0,
+    "max": "1",
+    "type": "Period"
+  } ],
+  "dataRequirement": [ {
+    "type": "Patient",
+    "profile": [ "http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-patient" ]
+  }, {
+    "type": "Encounter",
+    "profile": [ "http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-encounter" ],
+    "mustSupport": [ "type", "status", "status.value", "period" ],
+    "codeFilter": [ {
+      "path": "type",
+      "valueSet": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1111.143"
+    }, {
+      "path": "status.value",
+      "code": [ {
+        "code": "finished"
+      } ]
+    } ]
+  }, {
+    "type": "Encounter",
+    "profile": [ "http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-encounter" ],
+    "mustSupport": [ "type", "status", "status.value", "period" ],
+    "codeFilter": [ {
+      "path": "type",
+      "valueSet": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.3.117.1.7.1.292"
+    }, {
+      "path": "status.value",
+      "code": [ {
+        "code": "finished"
+      } ]
+    } ],
+    "dateFilter": [ {
+      "path": "period",
+      "valuePeriod": {
+        "extension": [ {
+          "url": "http://hl7.org/fhir/uv/crmi-analysisException",
+          "valueString": "Error attempting to determine filter value: toFhirValue not implemented for Subtract"
+        } ]
+      }
+    } ]
+  }, {
+    "type": "Encounter",
+    "profile": [ "http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-encounter" ],
+    "mustSupport": [ "type", "period", "status", "status.value" ],
+    "codeFilter": [ {
+      "path": "type",
+      "valueSet": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.3.666.5.307"
+    }, {
+      "path": "status.value",
+      "code": [ {
+        "code": "finished"
```

Comment:
Just a note that the IG publisher will generate a warning for a code without a system.

Reply:
This is a good and fair point, but without access to the structure definitions, I have no way of resolving what the system is... can you submit an issue to address this generally? It should be part of converting from data requirements to FHIR, but it will take resolving structure defs to do it, so beyond the scope of this PR